### PR TITLE
chore: badges

### PR DESCRIPTION
**Pull request by brianluong**
# Description

badges
## Type of change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] 😎 New feature (non-breaking change which adds functionality)
- [ ] ⁉️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] ⚒️ Refactor (no functional changes)
- [x] 📖 Documentation (updating or adding docs)

# How Has This Been Tested?
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # ⛩ node-gateway
+[![Test](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml/badge.svg)](https://github.com/satsuma-xyz/node-gateway/actions/workflows/test.yml) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/satsuma-xyz/node-gateway)](https://github.com/satsuma-xyz/node-gateway/releases) [![Docker Image Version (latest by date)](https://img.shields.io/docker/v/satsumaxyz/node-gateway?logo=docker)](https://hub.docker.com/r/satsumaxyz/node-gateway/tags) [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/satsuma-xyz/node-gateway/issues) [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/satsuma-xyz/node-gateway/blob/main/LICENSE)
```
Comment (dianwen) [inline README.md:2]: Could we have each badge on its own line for readability?

**Review by dianwen — APPROVED** [review]
Looks snazzy

File: README.md (hunk 1 above)
Comment (brianluong) [reply README.md]: IMO think it's too long to have one badge every line. How about batching them in a line by category? 

1. CI/CD
2. Releases
3. Contributions


Looks like this:
image

Comment (dianwen) [reply README.md]: Sorry, I meant in the code. Is there a way to have separate lines in code but same line in markdown?

Comment (brianluong) [reply README.md]: oh yeah you can!

Comment (jonathankau) [reply README.md]: @brianluong Mind making this tweak? Looks pretty odd to have multi line badges.

Comment (brianluong) [reply README.md]: yeah sure!

Comment (brianluong) [reply README.md]: https://github.com/satsuma-xyz/node-gateway/pull/57

**Pull request merged**